client/webserver/site/src/js/app.js

```diff
@@ -208,6 +229,31 @@ export default class Application {
     bind(pg.profileSignout, 'click', async e => await this.signOut())
   }
 
+  /*
+   * bindInternalNavigation hijacks navigation by click on any local links that
+   * are descendents of ancestor.
+   * */
+  bindInternalNavigation (ancestor) {
```

Comment:
Using hyperlinks for internal navigation allows the user to open the view in a new tab or window using the browsers shortcut key or context menu.